docs/modules/ROOT/pages/release-notes/release-notes-2020-7.adoc

```diff
@@ -0,0 +1,79 @@
+= Payara Community 5.2020.7 Release Notes
+
+== Supported APIs and Applications
+
+* Jakarta EE 8
+* Java EE 8 Applications
+
+== Notes
+
+=== MicroProfile 4.0
+
+As MicroProfile moves toward the 4.0 milestone, the Payara team has simultaneously been working to ready the Payara Platform for MicroProfile 4.0 compatibility. In this Payara Platform Community Release, you can give two of the MicroProfile specification release candidates a try: MP Health 3.0 and MP JWT Auth 1.2. This technically breaks MicroProfile 3.3 support, although functionally the MicroProfile 3.3 TCK will only fail on the `@Health` annotation removal.
```

Comment:
`As MicroProfile moves toward the 4.0 milestone` should be
`As MicroProfile moves toward the 4.0 final release, the Payara team ...`